src/hotspot/os/linux/os_linux.cpp

```diff
@@ -5134,10 +5130,11 @@ static jlong slow_thread_cpu_time(Thread *thread, bool user_sys_cpu_time) {
                  &ldummy, &ldummy, &ldummy, &ldummy, &ldummy,
                  &user_time, &sys_time);
   if (count != 13) return -1;
+  double clock_tics_factor = 1000000000 / os::Posix::clock_tics_per_second();
```

Comment:
I think you should rather not add clock_tics_factor but replace clock_tics_per_sec with os::Posix::clock_tics_per_second() in place, but it's maybe just a matter of taste...

Reply:
Okay I change it.